src/components/edit/exchange-update/getExchangeValues.js

```diff
@@ -4,15 +4,38 @@ import {
     AUTHENTICATION_TYPES,
 } from '../shared/index.js'
 
-const removeRequestIDSchemeValuesOfNONE = ({ requests }) => {
+// this filters out additional properties that are used by the app UI, but should not be sent to backend
+const validProperties = [
+    'name',
+    'visualization',
+    'dx',
+    'pe',
+    'ou',
+    'filters',
+    'inputIdScheme',
+    'outputDataElementIdScheme',
+    'outputOrgUnitIdScheme',
+    'outputIdScheme',
+    'outputDataItemIdScheme',
+]
+const copyRequestWithValidProperties = (request) =>
+    validProperties.reduce((requestCopy, prop) => {
+        if (request[prop]) {
+            requestCopy[prop] = request[prop]
+        }
+        return requestCopy
+    }, {})
+
+export const cleanUpRequests = ({ requests }) => {
     const idSchemeProps = [
-        'idScheme',
-        'dataElementIdScheme',
-        'orgUnitIdScheme',
-        'categoryOptionComboIdScheme',
+        'outputDataElementIdScheme',
```

Comment:
this was the source of the error before. I had the wrong names for the ID schemes relevant for requests.